src/user/index.js

```diff
@@ -3,11 +3,15 @@ const list = require('./list')
 const update = require('./update')
 const deleteUser = require('./delete')
 const notifications = require('./notifications')
+const authorizeDeviceSubscriptions = require('./authorizeDeviceSubscriptions')
+const authorizeDeviceProcessSubscriptions = require('./authorizeDeviceProcessSubscriptions')
 
 module.exports = {
   get,
   list,
   update,
   delete: deleteUser,
   notifications,
+  authorizeDeviceSubscriptions,
+  authorizeDeviceProcessSubscriptions,
 }
```

Comment:
nexus is attempting to nest resources, e.g.:

`device.certificate.get`
`log.device.deleteMany`
`user.notifications.putAll`

understanding that, is there a possible nested home for these that makes sense? spitballing some ideas:

(everywhere where `/create()` or is written could be anything like `/create()` | `/createMany()` | `authorize()` | `authorizeMany()` or other possibilities)

A. (understanding everything is user authorized already, except the couple public endpoints, so not under the top level `user`)
`device.subscription.create()` || `device.subscribe()`
`device.processes.subscriptions.create()` || `device.processes.subscribe()`

B. (explicit about user specific)
`user.device.subscriptions.create()` || `user.device.subscribe()`
`user.device.process.create()`  || `user.device.processes.subscribe()`

C. (in a bounded context meant only for real time subscriptions)
`subscription(s).device.create()`
`subscription(s).device.process.create()`

D. (...there are surely other options)

This is a big comment, but the question is short: is there a place where these methods could be nested that follows the pattern elsewhere/sets up additional subscriptions in the future to have obvious homes, OR is flat in the `user` resource the best place?

(dataPoints got away from this nesting, they could also be nested in the same pattern via `process.functionality.datapoints.get` or something, but only mentioning that to illustrate that it is not a comparison for the standard if looking at the file tree)

Reply:
fixed